tools/verify_translations.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/python3
+# Copyright (c) 2017-2023, Mudita Sp. z.o.o. All rights reserved.
+# For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+import collections
+import os
+import os.path as path
+import argparse
+import json
+import sys
+import logging
+import textwrap
+
+logging.basicConfig(level=logging.DEBUG, format='%(levelname)s: %(message)s')
+logger = logging.getLogger(__name__)
+handler = logging.StreamHandler(sys.stdout)
+logger.addHandler(handler)
+
+
+def detect_duplicate_keys(list_of_pairs: list):
+    key_count = collections.Counter(k for k, v in list_of_pairs)
+    duplicate_keys = ', '.join(k for k, v in key_count.items() if v > 1)
+
+    if duplicate_keys:
+        raise ValueError(duplicate_keys)
+
+
+def validate_data(list_of_pairs: list):
+    detect_duplicate_keys(list_of_pairs)
+    # More detection, each of them will raise exception upon invalid
+    # data
+    return dict(list_of_pairs)
+
+
+def perform_on_files_from_path(json_path: path, operation):
+    dir_list = os.listdir(json_path)
+    ret = 0
+
+    for file in dir_list:
+        file_path = path.join(json_path, file)
```

Comment:
If you import `from pathlib import Path` and use `Path` you would be able to replace `path.join` with simple `+ operator`.

Reply:
it works, but I wanted to leave it in a old-fashioned way :P